Assets/Scripts/Weapons/TripleshotWeapon.cs

```diff
@@ -30,6 +30,7 @@ public override void Attack(string tag, GameObject target)
             // Spawn the forward-facing projectile into the scene
             GameObject projectileInstance = Instantiate(projectile, spawnForwardPos, transform.rotation);
             projectileInstance.tag = tag;
+            projectileInstance.layer = LayerMask.NameToLayer(tag);
```

Comment:
Maybe this would be easier to put in the projectile's `Start` method?

Reply:
But the projectile would somehow have to know what layer it needs to be on.